CONTRIBUTING.md

```diff
@@ -0,0 +1,188 @@
+# CONTRIBUTING
+
+## Resources
+
+If you wish to contribute to Disco, please be sure to
+read/subscribe to the following resources:
+
+ -  [The PSR-2 Coding Standard](http://www.php-fig.org/psr/psr-2/)
+ -  The Disco Contributor's IRC channel: #disco.dev on Freenode.net
+
+If you are working on new features or refactoring [create a proposal](https://github.com/bitExpert/disco/issues/new).
+
+## Reporting Potential Security Issues
+
+If you have encountered a potential security vulnerability, please **DO NOT** report it on the public issue tracker: send it to us at [opensource@bitexpert.de](mailto:opensource@bitexpert.de) instead.
+We will work with you to verify the vulnerability and patch it as soon as possible.
+
+When reporting issues, please provide the following information:
+
+- Component(s) affected
+- A description indicating how to reproduce the issue
+- A summary of the security vulnerability and impact
+
+We request that you contact us via the email address above and give the project contributors a chance to resolve the vulnerability and issue a new release prior to any public exposure.
+This helps protect users and provides them with a chance to upgrade and/or update in order to protect their applications.
+
+For sensitive email communications, please use [our PGP key](http://pgp.mit.edu/pks/lookup?op=vindex&search=0xEC37D631657E0A89).
+
+## Running Tests
+
+To run tests:
+
+1. Clone the repository:
+
+```console
+$ git clone git@github.com:bitExpert/disco.git
+$ cd disco
+```
+
+2. Install dependencies via composer:
+
+```console
+$ curl -sS https://getcomposer.org/installer | php --
```

Comment:
Not sure if it makes sense to better refer to the Composer documentation on how to [install Composer](https://getcomposer.org/download/) as their docs mention the more "secure" method.